.devops/code-review-pipelines.yml

```
@@ -57,4 +53,16 @@ steps:
   - task: SonarCloudPublish@1
     displayName: 'Publish SonarCloud results on build summary'
     inputs:
-      pollingTimeoutSec: '300'
\ No newline at end of file
+      pollingTimeoutSec: '300'
+
+  - script: |
+      docker-compose -f docker-compose-springboot-postgres.yaml up --build -d
+    displayName: 'Run app with docker-compose'
+
+  - script: |
+      timeout 180 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:8085/healthcheck)" != "200" ]]; do sleep 5; done' || exit 1
+    displayName: 'Health check'
+
+  - script: |
+      docker ps --filter status=exited --format="{{.Image}} :: {{.Status}}" | grep liquibase | cut -d':' -f3 | sed -r "s/Exited \((.*)\) .*/\1/g" | tr -d ' '
```

Fix: this propagates status code to the task 😃

```suggestion
      [[ "$(docker ps --filter status=exited --format="{{.Image}} :: {{.Status}}" | grep liquibase | cut -d':' -f3 | sed -r "s/Exited \((.*)\) .*/\1/g" | tr -d ' ')" == "0" ]]
```